tests/test_models.py

```diff
@@ -1,28 +1,33 @@
 import json
+import logging
 import os
 
 from asnake.client import ASnakeClient
 import pytest
 import requests_mock
+import structlog
 
 from asaps import models
 
 
+logger = structlog.get_logger()
```

Comment:
You shouldn't need this here since you aren't actually using the logger to log anything in this test module.